### PR TITLE
Feature/openshift

### PR DESCRIPTION
**Pull request by goern**
This PR enabled Currentweather to be deployed to OpenShift, via a template, to be used by `oc new-app`

### CODE REVIEW

File: openshift/currentweather-template.yaml [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+      name: "FRONTEND_SOURCE_REPOSITORY_REF"
+      description: "FRONTEND: Set this to a branch name, tag or other ref of your repository if you are not using the default branch"
+      value: "feature/more-ui"
+    -
+      name: "FRONTEND_CONTEXT_DIR"
+      description: "FRONTEND: Set this to the relative path to your project if it is not in the root of your repository"
+    -
+      name: "REDIS_SERVICE_NAME"
+      description: "redis service name"
+      value: "currentweather-redis"
```
Comment (luebken) [inline openshift/currentweather-template.yaml]: TODO change to "redis"

File: openshift/currentweather-template.yaml [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+        dockerImageRepository: "goern/s2i-nginx"
+      tags:
+        - name: latest
+    - kind: "ImageStream"
+      apiVersion: v1
+      metadata:
+        description: "This is redis 2.8"
+        name: redis
+      spec:
+        dockerImageRepository: goern/redis-28-centos7
```
Comment (luebken) [inline openshift/currentweather-template.yaml]: TODO: Why not redis?

Comment (goern) [inline openshift/currentweather-template.yaml]: updated in readme

File: openshift/currentweather-template.yaml (hunk 1 above)
Comment (goern) [inline openshift/currentweather-template.yaml]: fixed...

**Pull request merged**